.githooks/pre-commit

```diff
@@ -1,3 +1,7 @@
+#!/bin/sh
+echo "executing post-commit"
+exit 0
```

Comment:
However... We shouldn't quit right here in the beginning! You've just disabled the hook :(
```suggestion
```

Reply:
Oh, my fail. Now it seems like I improved it.